docs/cloud/aws.md

```diff
@@ -38,34 +39,57 @@ If you want to use a specific profile, set `AWS_PROFILE`
+Use `--discover-filter` to add filters for regions, instance ids, and tags:
 
 ```bash
-$ mondoo scan -t ec2://profile/name/region/us-east-1
-$ mondoo scan -t ec2://region/us-east-1
-$ mondoo scan -t ec2://user/ec2-user
-$ mondoo scan -t ec2://profile/mondoo/region/us-east-1/user/ec2-user
+$ mondoo scan -t aws:// --discover all --discover-filter regions=us-east-2 --discover-filter instance-ids=i-06eab6c104c0f5fb0 --discover-filter tags=Name:testnametag
 ```
 
-![Mondoo AWS EC2 instances scan from CLI](../static/videos/aws-ec2-scan.gif)
+## Mondoo-AWS Integration
 
-> Note: mondoo uses `~/.ssh/config` to determine the users for each detected public IP
+Use the AWS integration to enable cron-scheduled and event-based scanning of your AWS account and EC2 instances.
 
-Instead of using the same ssh username for all instances, you can also configure the SSH config and configure the username for each instance with their username:
+### How it works 
+We use AWS Cloudformation to setup the Mondoo Lambda function in the desired account. The lambda function scans the account and EC2 instances on the desired cron schedule. It communicates with Mondoo Cloud to find the appropriate policies and sends all results to Mondoo Cloud.
 
-```bash
-Host 123.123.123.123
-  User chris
+### Create
+Use the Mondoo UI to create the integration:
+
+![integration-create-image](../../assets/aws-integration-create.png)
+
+After entering your AWS Account ID, the browser will open a new tab with AWS Cloudformation loaded, ready to launch the stack. 
+
+Scan Configuration options, explained:
+- cron scan in hours: the interval at which scans should be executed
+- discover ec2 instances: find the ec2 instances in the account (across all regions)
+- use ssm: use ssm to scan the instance when available 
+- use ssh: use ssh to scan instances, using the secrets metadata query and aws secrets manager vault to retrieve the secrets
+- secrets metadata query: see [Mondoo vault docs](../getstarted/vault.md)
+- show ec2 instances filtering options: displays fields to enter in region, instance id, or tag filters. this mirrors the cli `--discover-filter` behaviour noted above
+
+### View
+A list of integrations and a detail page for each is available on the agents page:
+
+![integration-list-image](../../assets/aws-integration-list.png)
+
+
+Clicking on any of the integration rows will lead to the details page for that integration. Any errors the server encountered when setting up the integration will be displayed on the details page. This is also where scan configuration options can be edited and a one-off scan triggered.
+
+
+### Remove
+When removing an AWS Integration from Mondoo server, a link will be opened to the AWS Cloudformation Stacks list page for easy stack deletion. The integration information will be deleted from the Mondoo database, and the rule allowing the Mondoo AWS account to send events to the target acccount will be deleted. 
```

Comment:
Maybe more like:

When removing an AWS Integration from Mondoo server, a link will be opened to the AWS Cloudformation Stacks list page for easy stack deletion. The configured integration will be removed from the Mondoo, and the rule allowing the Mondoo AWS account to send events to the target account will be deleted.